+nix/MultiTag.m

```diff
@@ -0,0 +1,93 @@
+classdef MultiTag < nix.Entity
+    %MultiTag nix MultiTag object
+
+    properties(Hidden)
+        info
+    end;
+    
+    properties(Dependent)
+        id
+        type
+        name
+        definition
+        units
+        featuresCount
+        sourcesCount
+        dataArrayReferenceCount
```

Comment:
imho, this property should rather be called referenceCount, that would be a bit easier.
similarly I would rather rename featuresCount to featureCount